src/main/java/mate/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,42 @@
+package mate.filter;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+@WebFilter(urlPatterns = "/*")
+public class AuthenticationFilter implements Filter {
+    private static Set<String> URL = new HashSet<>();
```

Comment:
this field is not constant so we can name it regular way
```suggestion
    private static Set<String> allowedUrls = new HashSet<>();
```